orders/tests.py

```diff
@@ -52,3 +52,14 @@ def test_dog_owner_id_is_deleted_when_dog_owner_is_deleted(self, create_order):
     def test_daycare_id_is_deleted_when_daycare_is_deleted(self, create_order):
         DayCare.objects.get(id=1).delete()
         assert Order.objects.get(id=create_order.id).daycare_id is None
+
+    def test_get_all_day_cares_available_on_dates_function(self):
+        start_date = "2022-05-02"
+        end_date = "2022-08-02"
```

Comment:
seems like a classic case for a `parametrize` test to me.

Reply:
Fixed.